src/test/lrsql/lrs_test.clj

```diff
@@ -367,6 +403,42 @@
     (component/stop sys')
     (support/unstrument-lrsql)))
 
+(deftest attachment-normalization-test
+  (let [sys   (support/test-system)
+        sys'  (component/start sys)
+        lrs   (-> sys' :lrs)
+        id-4  (get stmt-4 "id")
+        id-5  (get stmt-5 "id")]
+
+    (testing "accepts normalized attachments"
+      (is (= {:statement-ids [id-4
+                              id-5]}
+             (lrsp/-store-statements
+              ;; stmt-5 references stmt-4-attach AND stmt-5-attach
+              lrs auth-ident [stmt-4 stmt-5] [stmt-4-attach stmt-5-attach]))))
+
+    (testing "returns normalized attachments"
```

Comment:
meh, it was pretty much covering that, as both statements share stmt-4-attach, but sure: https://github.com/yetanalytics/lrsql/pull/225/commits/6e98b8f99988e8b983197c76613567e03befb49f

Reply:
Yeah, but I thought it would be more immediately obvious if the number of statements was strictly greater than the number of attachments.

(Also that `stmt-5 references...` comment now needs to be updated.)

Reply:
https://github.com/yetanalytics/lrsql/pull/225/commits/bee9ec2ff232a3cce3489502a10519cfd7ebe425